packages/opentelemetry-cloud-monitoring-exporter/src/monitoring.ts

```diff
@@ -92,14 +93,38 @@ export class MetricExporter implements IMetricExporter {
     metrics: MetricRecord[],
     cb: (result: ExportResult) => void
   ): Promise<void> {
+    return this._exportAsync(metrics)
+      .then(cb)
+      .catch(err => {
+        diag.error(err.message);
+        return cb({code: ExportResultCode.FAILED, error: err});
+      });
+  }
+
+  async shutdown(): Promise<void> {}
+
+  /**
+   * Asnyc wrapper for the {@link export} implementation.
+   * Saves the current values of all exported {@link MetricRecord}s so that
+   * they can be pulled by the Google Cloud Monitoring backend.
```

Comment:
Just noticed this original description doesn't really make sense, there is no pulling. Can we just make it "Writes MetricRecords to Cloud Monitoring" or similar?